### ISSUE TITLE
Library of typical elements 

### ISSUE THREAD

**Issue by Isaina** [krzysztofarendt/building3d#114]
Hi,

can you make 3D element to put into building like, wall Light, ceiling light , sound bar, aircon indoor , outdoor unit ,  TV, Radio, Laptop,  Computer for the for egg. Light we can tune up or change lux parameters for sound bar or Tv , sound pressure or sound power in dB, so we can put this models in 3d environment

**Comment by krzysztofarendt** [comment]
Yes, this can be done. But it is also already possible to import external STL files.

I would like to finish the sound simulation engine first. There is still a couple things to be done, like frequency dependent reflections.
